plugin/environment.go

```diff
@@ -238,6 +236,9 @@ func (env *Environment) Activate(id string) (manifest *model.Manifest, activated
 	// Store latest BundleInfo in case something has changed since last activation
 	rp.BundleInfo = pluginInfo
 	env.registeredPlugins.Store(id, rp)
+	if env.pluginHealthCheckJob != nil {
+		env.pluginHealthCheckJob.EnsurePlugin(id)
```

Comment:
do we need to register plugins to the healthchecker? I wonder if we can rely on querying `job.env.Active()` on every heartbeat and store the failed plugins inside the healthchecker? 🤔